### PR TITLE
Sites Dashboard v2: Update styles for GSV

### PR DESCRIPTION
**Pull request by candy02058912**
Fixes https://github.com/Automattic/dotcom-forge/issues/6714
Fixes https://github.com/Automattic/dotcom-forge/issues/6754

## Proposed Changes

-  Update header font styles
-  Update link styles (color and size)
-  Adjust site icon size
-  Add box-shadow to site icon

Figma: PZtnZ1nHhnBrsuXDoMJHPD-fi-446_63469

| Before  | After |
| :-------------: | :-------------: |
|   | |

## Testing Instructions

* Go to /sites?flags=layout/dotcom-nav-redesign-v2
* Make sure the styles are aligned with Figma
* Test on mobile to see if there's anything weird

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by cleacos — CHANGES_REQUESTED** [review]
Sorry... This one is touching the ItemPreviewPane components.

I'll unlock it after merging the PR I'm working on with changes.

File: client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
 				<div className={ classNames( 'site-favicon', className ) }>
-					<SiteIcon size={ size } defaultIcon={ defaultFavicon } />
+					<SiteIcon
+						size={
+							isLargerThan960px
+								? siteIconSize?.default || defaultSiteIconSize.default
+								: siteIconSize?.small || defaultSiteIconSize.small
+						}
+						defaultIcon={ defaultFavicon }
+					/>
```
Comment (cleacos) [inline client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx]: This was updated. Already in trunk

File: client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx [hunk 2]
```diff
@@ -42,8 +47,22 @@ export default function ItemPreviewPaneHeader( {
...
+						{ itemData.adminUrl && (
+							<Button
+								variant="link"
+								className="site-preview__header-summary-link"
+								href={ `${ itemData.adminUrl }` }
+								target="_blank"
+							>
+								<span>{ translate( 'WP Admin' ) }</span>
+							</Button>
+						) }
```
Comment (cleacos) [inline client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx]: Updated. Check trunk

File: client/a8c-for-agencies/components/items-dashboard/item-preview-pane/types.ts [hunk 3]
```diff
@@ -22,14 +22,26 @@ export interface ItemData {
 	url?: string;
 	icon?: string;
 	color?: string;
+	adminUrl?: string;
```
Comment (cleacos) [inline client/a8c-for-agencies/components/items-dashboard/item-preview-pane/types.ts:28]: Updated. Check trunk

**Review by cleacos — COMMENTED** [review]
Check the conflicts... but probably it will be better to start fresh, and cherry-pick your code.

**Review by taipeicoder — APPROVED** [review]
Code LGTM 👍 
Some non-blocking notes on my end:

1. The close button position is not as in Figma.

| Figma | PR |
| --- | --- |
| Screenshot 2024-04-26 at 4 33 36 PM |  Screenshot 2024-04-26 at 4 34 01 PM| 

2. The external icon in Figma is 16x16, but in the PR seems to be 24x24.
3. In smaller viewports, the text doesn't seem vertically aligned and the icons too big in comparison with the text. 
Screenshot 2024-04-26 at 4 36 09 PM

**Comment by candy02058912** [comment]
Will update the external icon link, went missing after the rebase.

**Comment by allilevine** [comment]
> The external icon in Figma is 16x16, but in the PR seems to be 24x24.

It looks like the icon is now 16x16, but it's not aligned with the text:

Screen Shot 2024-04-26 at 11 58 42 AM

Figma:

Screen Shot 2024-04-26 at 11 59 56 AM

Non-blocking, as noted above, but we might want to make it slightly larger.

**Comment by allilevine** [comment]
> 1. The close button position is not as in Figma.

I opened a follow-up: https://github.com/Automattic/dotcom-forge/issues/6772

**Review by cleacos — APPROVED** [review]
At the A4A side it looks good!

image
At UI/design level, is the WP Admin link okay? it looks weird... font size...

LGTM!

**Review by DustyReagan — APPROVED** [review]
Nice work! 🎉  I'll go ahead and merge this one and we can followup on the remaining styling tweaks.

**Pull request merged**